CrewChiefV4/Audio/Sounds.cs

```diff
@@ -21,18 +21,19 @@ public class SoundCache
         
         private List<String> dynamicLoadedSounds = new List<String>();
         public static Dictionary<String, SoundSet> soundSets = new Dictionary<String, SoundSet>();
-        private Dictionary<String, SingleSound> singleSounds = new Dictionary<String, SingleSound>();
+        private static Dictionary<String, SingleSound> singleSounds = new Dictionary<String, SingleSound>();
         public static List<String> sortedAvailableDriverNames = new List<String>();
         public static List<String> sortedAvailableSounds = new List<String>();
         public static List<String> sortedAvailablePrefixesAndSuffixes = new List<String>();
```

Comment:
just re-instated this so if load_sound_files_on_startup is false we don't read the files until they're played